### PR TITLE
Update platform version 0.31.0-alpha.0

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Signed-off-by: Neeharika-Sompalli <neeharika.sompalli@swirldslabs.com>

### CODE REVIEW

**Review by nathanklick — APPROVED** [review]

File: hedera-node/configuration/preprod/bootstrap.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ bootstrap.throttleDefsJson.resource=throttles.json
 contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
-hedera.recordStream.enableTraceabilityMigration=true
\ No newline at end of file
+hedera.recordStream.enableTraceabilityMigration=false
```
Comment (nathanklick) [inline hedera-node/configuration/preprod/bootstrap.properties:5]: Is this related or should be changed in this release?

**Review by nathanklick — APPROVED** [review]

File: hedera-node/configuration/preprod/bootstrap.properties (hunk 1 above)
Comment (Neeharika-Sompalli) [reply hedera-node/configuration/preprod/bootstrap.properties:5]: yeah , should be changed back. Confirmed with @SimiHunjan
